pyvat/countries.py

```diff
@@ -10,6 +10,7 @@
     'ES',  # Spain.
     'FI',  # Finland.
     'FR',  # France.
+    'GB',  # United Kingdom.
```

Comment:
Shouldn't this say "Great Britain" ? There is some difference: https://www.dictionary.com/e/great-britain-vs-uk-vs-england/#:~:text=Quick%20summary,Scotland%2C%20and%20Wales%20are%20situated.

Reply:
Set the comment to United Kingdom because it was that on version 1.3.6, felt it made sense to keep it the same. 
The API accepts a UK VAT number so maybe it should be UK instead of GB.

https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/vat-registered-companies-api/1.0